### ISSUE TITLE
Destroy not working as expected

### ISSUE THREAD

**Issue by thomasfrobieter** [somewebmedia/hc-sticky#35]
After fire the destroy command, the sidebar still has the inline css styles:

width: 100px;
top: 0px;
bottom: 0px;
left: 0px;
right: 717px;
position: absolute;

This should be removed also?

**Comment by askibinski** [comment]
Yes, noticed that too. Our workaround was to manually remove those inline styles, but I think this should be fixed in the script.
